app/src/main/java/com/rentminder/SaveButton.kt

```diff
@@ -35,17 +35,11 @@ import androidx.compose.ui.unit.dp
 fun SaveRemindButton(enabled: Boolean, inputEdited: MutableState<Boolean>) {
     val context = LocalContext.current
     val focusManager = LocalFocusManager.current
-    var count by remember { mutableStateOf(0) }
 
     IconButton(
         onClick = {
             if (enabled) {
-                if (count > 0) {
-                    Toast.makeText(context, "Edit Saved!", Toast.LENGTH_SHORT).show()
-                } else {
-                    Toast.makeText(context, "Saved!", Toast.LENGTH_SHORT).show()
-                }
-                count++
```

Comment:
Avoid using variables that can cause recompositions. In particular, when you use count++, you give the opportunity for the composer to infinitely recompose in a loop which is not good. For this reason, I believe that having the count variable is redundant and should be removed. See this link for reference: https://developer.android.com/jetpack/compose/performance/bestpractices#avoid-backwards

Reply:
Good point.  A few other thoughts:
- There's not a whoooooole lot of difference between "Edit Saved" and "Saved" messaging.  "Saved" is fine.
- Do you need to toast when saving?  Maybe not.  That's not a common paradigm anymore, as a) it's distracting, and b) many documents auto-save now, and allow you to go back in history.  If you do want to indicate saved, consider less intrusive messaging, like changing the background color and/or vibrating the device.
- Move strings to strings.xml for internationalization.